test/test_torch.py

```diff
@@ -15956,7 +15956,10 @@ def test_eig(self, device, dtype):
         self.assertEqual(vv, vvv, atol=1e-12, rtol=0)
         self.assertEqual(vv, tv, atol=1e-12, rtol=0)
 
-        # test reuse
+    @onlyCPU
+    @skipCPUIfNoLapack
+    @dtypes(torch.double)
+    def test_eig_reuse(self, device, dtype):
```

Comment:
Is there a test that (1) we can run in float precision, too, and (2) we can use a NumPy's eig as a reference? Maybe not by comparing the functions directly but, since the functions compute the same thing, validating the computations are close?

Reply:
done by ca7f4991e5 (and 0f8148ab41 for an additional comment): I enabled `torch.float` for all eig tests (because, why not?)
I had to use an rtol!=0 for a few torch.float tests when comparing the results, though.